vignettes/setup.Rmd

```diff
@@ -0,0 +1,133 @@
+---
```

Comment:
Nice! I'll likely amend this slightly at some point but it reads well already! Thank you!